src/pages/login/login.tsx

```diff
@@ -16,6 +18,7 @@ const Login: FC<LoginProps> = ({ history }) => {
 			setInvalid(false);
 			localStorage.setItem('admin', 'Solomon');
```

Comment:
Instead of using a hard coded value like solomon here, let it be based on the username. Incase you have multiple accounts, they'll all show solomon when logged instead of the right username.